src/Endpoints/Delegates/HandlesMultiSearch.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Meilisearch\Endpoints\Delegates;
+
+trait HandlesMultiSearch
+{
+    public function multiSearch(array $queries = [])
```

Comment:
```suggestion
{
    /**
     * @param list<SearchQuery> $queries
     */
    public function multiSearch(array $queries = [])
```

people with static analysis will thank you :)

Reply:
also `array<SearchQuery>` would also work

Reply:
Do you know how can I run this locally without phpstorm?

Reply:
@brunoocasali sorry missed this comment, not sure what you meant :)